### ISSUE TITLE
aura -Syuu said "Invalid option `-u'"

### ISSUE THREAD

**Issue by 0xBADDCAFE** [fosskers/aura#557]
pacman can specify `-Syuu` option about below:

```
  -u, --sysupgrade     upgrade installed packages (-uu enables downgrades)
```

I always use aura instead of pacman, so useful to be able to specify the option.

```
$  aura --version

 __ _ _  _ _ _ __ _    Pacman v5.2.1 - libalpm v12.0.1
/ _` | || | '_/ _` |   Copyright (C) 2006-2019 Pacman Development Team
\__,_|\_,_|_| \__,_|   Copyright (C) 2002-2006 Judd Vinet
AURA Version 2.0.4
 by Colin Woodbury     This program may be freely redistributed under
                       the terms of the GNU General Public License.
Aura Translators:
 (Japanese) Onoue Takuro
 (Polish) Chris Warrick
 (Croatian) Denis Kasak / "stranac"
 (Swedish) Fredrik Haikarainen / Daniel Beecham
 (German) Lukas Niederbremer / Jonas Platte
 (Spanish) Alejandro Gómez / Sergio Conde / Max Ferrer
 (Portuguese) Henry Kupty / Thiago Perrotta / Wagner Amaral
 (French) Ma Jiehong / Fabien Dubosson
 (Russian) Kyrylo Silin / Alexey Kotlyarov
 (Italian) Bob Valantin
 (Serbian) Filip Brcic
 (Norwegian) "chinatsun"
 (Indonesian) "pak tua Greg"
 (Chinese) Kai Zhang
 (Esperanto) Zachary Matthews
```

**Comment by fontmaniac** [comment]
Possibly related, after the recent upgrade `aura -Scc` reports "Invalid option `-c'"

**Comment by fosskers** [comment]
Likely related, yes. @fontmaniac for now you can consider the `-C` command that Aura provides.

**Comment by fosskers** [comment]
Fixed. I'm making a release with this included now.

**Comment by fosskers** [comment]
Released as `2.0.5`.

**Comment by 0xBADDCAFE** [comment]
Upgrade `2.0.5` and seems working. Thanks!
